media-libs/jbigkit/jbigkit-2.1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A lossless image compression library"
+DESCRIPTION="JBIG-KIT is a software implementation of the JBIG1 data \
+compression standard (ITU-T T.82), which was designed for bi-level image \
+data, such as scanned documents. This library is available in portable C \
+code. It is widely used in fax products, printer firmware, printer drivers, \
+document management systems and imaging software."
+HOMEPAGE="https://www.cl.cam.ac.uk/~mgk25/jbigkit/"
+COPYRIGHT="1995-2014 Markus Kuhn
+	1989, 1991 Jef Poskanzer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://www.cl.cam.ac.uk/~mgk25/download/jbigkit-$portVersion.tar.gz"
+CHECKSUM_SHA256="de7106b6bfaf495d6865c7dd7ac6ca1381bd12e0d81405ea81e7f2167263d932"
+PATCHES="jbigkit-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 ?x86"
+
+libVersion=$portVersion
+libVersionCompat="$libVersion compat >= 2"
+
+PROVIDES="
+	jbigkit$secondaryArchSuffix = $portVersion
+	cmd:jbgtopbm$secondaryArchSuffix
+	cmd:jbgtopbm85$secondaryArchSuffix
+	cmd:pbmtojbg$secondaryArchSuffix
+	cmd:pbmtojbg85$secondaryArchSuffix
+	lib:libjbig$secondaryArchSuffix = $libVersionCompat
+	lib:libjbig85$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	jbigkit_devel$secondaryArchSuffix = $portVersion
+	devel:libjbig$secondaryArchSuffix = $libVersionCompat
+	devel:libjbig85$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	jbigkit$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+defineDebugInfoPackage jbigkit$secondaryArchSuffix \
+	"$libDir"/libjbig.so \
+	"$libDir"/libjbig85.so
+
+BUILD()
+{
+	make all $jobArgs
+	cd libjbig
+	make -f Makefile_so $jobArgs
+}
+
+INSTALL()
+{
+	install -d -m 755 $binDir $includeDir $libDir $developLibDir
+
+	# install the tools
+	cd pbmtools
+	install -t "$binDir" -m 755 jbgtopbm{,85} pbmtojbg{,85}
+
+	# install headers and libs
+	cd ../libjbig
+	install -m 0644 -t "$includeDir" *.h
+	install -m 0644 -t "$libDir" libjbig*
+
+	prepareInstalledDevelLibs libjbig libjbig85
+
+	packageEntries devel \
+		$developLibDir \
+		$includeDir
+}
+
+TEST()
+{
+	cd libjbig
+	make test
```

Comment:
If you like using `make -C <directory>`, here's a neutral variant:
```diff
-	cd libjbig
-	make test
+	make -C test libjbig
```
(As said, this won't make any difference, so the only "advantage" of this variant is that you save a "cd" and a newline ;)